app/controllers/application_controller.rb

```diff
@@ -1,6 +1,4 @@
-class ApplicationController < ActionController::Base
-  protect_from_forgery with: :null_session
-
+class ApplicationController < ActionController::API
```

Comment:
Nice

Reply:
Merged last 2 pr's into this one, updated summary.